modules/datadict/ajax/UpdateDataDict.php

```diff
@@ -37,32 +37,32 @@
 } else {
     // Magic quotes is off, so we can just directly use the description
     // since insert() will use a prepared statement.
-    $description = $_REQUEST['description'];
+    $description = trim($_REQUEST['description']);
 }
 
 // create user object
 $user =& User::singleton();
 
 if ($user->hasPermission('data_dict_edit')) { //if user has edit permission
-    if ($DB->pselectOne(
-        "SELECT COUNT(*) FROM parameter_type_override " .
-        "WHERE Name =:id",
-        array('id' => $name)
-    )==0) {  //if it doesn't exist
-        // unsafeinsert is needed to allow '<TEST_NAME>_Date_taken' to be inserted
-        $DB->unsafeinsert(
+
```

Comment:
This prevent from inserting in parameter_type_overide when the description is not changed from the original and delete the record when it is back to the original